src/redux/reducers/loginReducer.js

```diff
@@ -0,0 +1,24 @@
+import ActionTypes from '../../constants/ActionTypes'
```

Comment:
that sounds like a cool idea, separate things by functional module. ill make changes for the Login page!